test_util/runner.py

```diff
@@ -44,9 +44,9 @@ def integration_test(
         'AWS_SECRET_ACCESS_KEY=' + aws_secret_access_key,
         'AWS_REGION=' + region]
 
-    pytest_cmd = """'source /opt/mesosphere/environment.export &&
+    pytest_cmd = """ "source /opt/mesosphere/environment.export &&
```

Comment:
This change has already been merged here:
https://github.com/dcos/dcos/commit/1a35a34b395f98527795f50a93acdc8af5702008